tests/testthat/test_tex_catwitherror.R

```diff
@@ -20,7 +20,7 @@ test_that('very_small_number', {
 
 test_that('same_error', {
     expect_equal(tex.catwitherror(0.00123, 0.00123, digits = 4, with.dollar = FALSE), '0.001230(1230)')
-    expect_equal(tex.catwitherror(12.345, 12.345, digits = 4, with.dollar = FALSE), '12.35(12.35)')
```

Comment:
why this change?

Reply:
Because the test fails otherwise. See above.

Reply:
Sorry, yes I missed that somehow (I was only looking at the changes). I think it's okay to round down .5, as .500000001 is rounded up.